### ISSUE TITLE
Mode 1 Lua Call Stack - Freezes all vehicles/player in game until StopAll command used.

### ISSUE THREAD

**Issue by cwattyeso** [Courseplay/courseplay#3823]
[log.txt](https://github.com/Courseplay/courseplay/files/3203114/log.txt)

Tried recording and running a course today with Courseplay in Mode 1 to transfer crops from one custom placed silo building on the map, to the original main farm silo. If I start the course with the vehicle already filled at the first waypoint it drives the course, unloads when it gets to the second silo, drive back to the first silo and as it gets near first waypoint/fill point it freezes and starts generating the Lua error in the log. This also causes all other vehicles on the map to freeze or begin driving erratically, the player also becomes frozen. Only way out of this issue is to bring up the Dev Console and use the StopAll command for Courseplay Workers to regain control of the player and vehicles on the map.

This is the first time I've got a problem with Mode 1, and loading other previously created and saved courses for the Main Silo doesn't produce any issues. So is it an issue specific to the Silo Building being used in this course?

**Comment by Tensuko** [comment]
Still Valid with Version 353 ?
I used Mode1 a lot the last days and couldn't create a LUA Callstack on that one.

**Comment by cwattyeso** [comment]
I haven't had the problem for a long time. However I suspect the problem when I opened this was due to the fact I was using a silo from the MF Shed Pack Mod on the Official ModHub https://www.farming-simulator.com/mod.php?lang=en&country=gb&mod_id=129738&title=fs2019

I've not used that particular mod with Mode 1 courses since reporting this problem. On many Mod Maps I've played since then, using the the silos put in by the map authors I've not had any Mode 1 issues. Nor have I experienced any with any Factory or Global Company Placeable Types buildings either.

So I'd guess the problem was more down to that Mod Pack and maybe the author needs to have a look at it and update it a bit.
